lib/experiments-page.php

```diff
@@ -52,6 +52,17 @@ function gutenberg_initialize_experiments_settings() {
 			'id'    => 'gutenberg-zoomed-out-view',
 		)
 	);
+	add_settings_field(
+		'gutenberg-off-canvas-navigation-editor',
+		__( 'Off canvas navigation editor ', 'gutenberg' ),
```

Comment:
I'm not sure about this name. I prefer Navigation list view, but I'm not 100%.

Reply:
Who cares :) It's gonna disappear anyway when the experiment is over.